app/models/issn_sanitizer.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class ISSNSanitizer
+  def initialize(value)
+    @value = value
+  end
+
+  def issn
+    return nil if ISBNSanitizer.new(value).isbn || DOISanitizer.new(value).url
+
+    /(ISSN |eISSN )?\S{4}-\S{4}/.match(value)
```

Comment:
I ran the AI import and found this regex was pulling things like this from urls: 'word-word'.

We should probably make this more restrictive with a regex like this: `(ISSN |eISSN )?\d{4}-\d{3}(\d|x|X)`